sonar-php-plugin/src/test/java/org/sonar/plugins/php/reports/phpstan/PhpStanSensorTest.java

```diff
@@ -68,10 +69,11 @@ public void test_descriptor() {
 
   @Test
   public void raise_issue() throws IOException {
-    List<ExternalIssue> externalIssues = executeSensorImporting("phpstan-report.json");
+    List<ExternalIssue> externalIssues = executeSensorImporting(reportPath("phpstan-report.json"));
     assertThat(externalIssues).hasSize(3);
 
-    ExternalIssue first = externalIssues.get(0);
+    List<Integer> orderOfIssues = SEPARATOR_CHAR == '\\' ? List.of(2, 0, 1) : List.of(0, 1, 2);
+    ExternalIssue first = externalIssues.get(orderOfIssues.get(0));
     assertThat(first.type()).isEqualTo(RuleType.CODE_SMELL);
```

Comment:
Not totally sure why the order of issue is not the same with Windows.
I guess it's coming from the fact that `executeSensorImporting()` is not parsing `phpstan-report.json` and `phpstan-report_win.json` the same way, resulting the issues order to be different ?
If that's right, from my understanding this is not a problem as the order of issue is not an important information, but this explain why you still had to adapt for this test.
How did you encounter this issue ? I guess if you are on a Unix system (I did understand I'm the only Windows user in the bubble :D ) , you didn't have it, so the error was raised by **build_win** CI step ?